test/server/cards/02_SHD/units/GentleGiant.spec.ts

```diff
@@ -0,0 +1,60 @@
+describe('Gentle Giant', function() {
+    integration(function(contextRef) {
+        describe('Gentle Giant\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [
+                            { card: 'gentle-giant', damage: 1 },
+                            { card: 'r2d2#ignoring-protocol', damage: 3 },
+                            { card: 'c3po#protocol-droid', damage: 1 }],
+                    },
+                    player2: {
+                        groundArena: [{ card: 'wampa', damage: 1 }]
+                    }
+                });
+            });
+
+            it('heal a target', function () {
```

Comment:
nit: can we add a bit more detail in the test case name in case someone unfamiliar with the card ends up needing to look at this test. Also note that the test name is concatenated between `describe()` and `it()`. So in this case the name would be: `Gentle Giant's ability heal a target.` Ideally it would be something like `Gentle Giant's ability can heal a uniot for the amount of damage it has`.

If you want to see the case name you can just force the test to fail